src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -188,6 +190,21 @@ public function isVariadic(): bool
         return false;
     }
 
+    /**
+     * Check if the function throw error/exception.
+     */
+    public function isThrowable(): bool
+    {
+        $visitor = new FindingVisitor(static function ($node) {
+            return $node instanceof NodeThrow;
+        });
+        $traverser = new NodeTraverser();
+        $traverser->addVisitor($visitor);
+        $traverser->traverse($this->getBodyAst());
+
+        return !empty($visitor->getFoundNodes());
```

Comment:
Let's avoid the usage of `empty`: `[] !== $visitor->getFoundNodes()` instead, since we know that it will be an `array`